WatchApp Extension/Extensions/CLKTextProvider+Compound.m

```diff
@@ -0,0 +1,33 @@
+//
+//  CLKTextProvider+Compound.m
+//  WatchApp Extension
+//
+//  Created by Michael Pangburn on 10/27/18.
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+#import "CLKTextProvider+Compound.h"
+
+// CLKTextProvider.textProviderWithFormat (compound text provider creation) is unavailable in Swift.
+// c.f. https://crunchybagel.com/using-multicolour-clktextprovider-in-swift-in-watchos-5/
+@implementation CLKTextProvider (Compound)
+
++ (CLKTextProvider *)textProviderByJoiningTextProviders: (nonnull NSArray<CLKTextProvider *> *)textProviders separator:(nullable NSString *) separator {
```

Comment:
Use NS_ASSUME_NONNULL_BEGIN / NS_ASSUME_NONNULL_END at the top and bottom of each file to streamline nullability annotations.